### ISSUE TITLE
Fix allowed colors for Alphanumeric and Shape submission in flaskapp

### ISSUE THREAD

**Issue by jmoxrox** [FlintHill/SUAS-Competition#88]
The drop down list for manual target submission doesn't have correct color choices. Here are the correct colors: ['white', 'black', 'gray', 'red', 'blue', 'green', 'yellow', 'purple', 'brown', 'orange']

**Comment by DarkmatterVale** [comment]
We need to verify that all of the options are able to be submitted. @GGJesus Can you please verify all of the submission options to make sure they are valid according to the interop submission specifications?

**Comment by jvillemare** [comment]
I talked with you in person and you asked me to specifically verify if:

- Shapes are correct, and
- Colors are correct (in the GCS:WebUI)

Here is the [AUVSI SUAS Shape and Color specs.](http://auvsi-suas-competition-interoperability-system.readthedocs.io/en/latest/specification.html#Shapes)

Here is what I did

- I checked and confirmed that **the selectable shapes are indeed all the possible shapes allowed by the Interoperability server.** I just took the list of shapes from the spec and sorted them alphabetically.
- I checked the colors and **found that they were, in fact, wrong.** I think that I just forgot the spec, and instead, assumed it was RoyGiBiv. I have corrected the mistake in https://github.com/FlintHill/SUAS-Competition/commit/ea1ed06c2188eeb08c16d6115bb0dafcc20c4000 for both color options.
